apisix/cli/ops.lua

```diff
@@ -821,6 +837,9 @@ local function start(env, ...)
         end
 
         print("Use customized yaml: ", customized_yaml)
+    else
+        -- Cleanup any custom config from previous runs
+        cleanup()
```

Comment:
I think we should not do this in every start, try to catch the error info of `util.execute_cmd(env.openresty_args)` and do the `cleanup`

Reply:
You mean to check the error of `util.execute_cmd(env.openresty_args)` and if the error is related to cleanup then cleanup and run `util.execute_cmd(env.openresty_args)` again?

Reply:
No need to run again, call the function `cleanup()`

Reply:
Okay understood

Reply:
@monkeyDluffy6017 This is an initialization error and doesn't come from the openresty process. So I think we should check the error of `init(env)` and do the cleanup in case of error. What do you think?

Reply:
ok